src/peoplemeasurement/serializers.py

```diff
@@ -41,13 +41,32 @@ class Meta:
         fields = '__all__'
 
 
-class AreaSerializer(serializers.ModelSerializer):
+class GeoSerializer(serializers.ModelSerializer):
+    sensor = serializers.SlugRelatedField(slug_field="objectnummer", queryset=Sensors.objects.all())
+
+    def get_validation_errors(self, errors=None) -> list[str]:
+        """
+        Returns full errors formatted as per requirements
+        """
+        default_errors = errors or self.errors  # default errors dict
+        error_messages = []
+        for field_name, field_errors in default_errors.items():
+            if isinstance(field_errors, list):
+                for error in field_errors:
+                    error_messages.append(f"{field_name}: {error}")  # append error message to 'errors_messages'
```

Comment:
Nitpick: Hier gebruik je append, maar hieronder gebruik je += is er een reden hiervoor? Daarnaast vind ik de commentaar overbodig.